sys/androidmedia/gstamc-constants.h

```diff
@@ -101,7 +101,10 @@ enum
   /* From hardware/ti/omap4xxx/domx/omx_core/inc/OMX_TI_IVCommon.h */
   COLOR_TI_FormatYUV420PackedSemiPlanarInterlaced = 0x7f000001,
   COLOR_EXYNOS_FormatNV12Tiled = 0x7fc00002,
-  COLOR_EXYNOS_FormatNV21Linear = 0x7f000011
+  COLOR_EXYNOS_FormatNV21Linear = 0x7f000011,
+  /* MTK formats */
```

Comment:
Further explain what these colors are for, that they were added for HEVC (h.265) and VP8/9 support. Can you explicitly label which one is for which format?